packit_service/worker/helpers/build/copr_build.py

```diff
@@ -392,7 +393,10 @@ def is_forge_project_allowed_to_build_in_copr(self) -> bool:
             self.job_project,
         )
         allowed_projects = copr_project["packit_forge_projects_allowed"]
-        allowed = self.forge_project in allowed_projects
+        allowed = any(
+            fnmatch.fnmatch(self.forge_project, pattern)
+            for pattern in allowed_projects.splitlines()
```

Comment:
we are receiving list here, so the `splitlines()` shouldn't be here:

```
>>> r = c.project_proxy.get("lbarczio", "test-repo")
>>> r["packit_forge_projects_allowed"]
"['github.com/the-project/*', 'github./com/the-project/the-namespace']"
```

Reply:
I'm afraid it's worse than that…

```
>>> config = {'copr_url': 'https://copr.fedorainfracloud.org'}
>>> c = Client(config=config)
>>> r = c.project_proxy.get("lbarczio", "test-repo")
>>> type(r["packit_forge_projects_allowed"])
<class 'str'>
>>> r["packit_forge_projects_allowed"]
"['github.com/the-project/*', 'github./com/the-project/the-namespace']"
```

It's a string that looks like a list!